stdlib/database/jdbc/src/main/ballerina/src/java.jdbc/Module.md

```diff
@@ -0,0 +1,92 @@
+## Module overview
+
+This module provides the functionality required to access and manipulate data stored in any type of relational database 
+that is accessible via Java Database Connectivity (JDBC). 
+
+**Prerequisite:** Add the JDBC driver corresponding to the database you are trying to interact with
+as a native library dependency in your Ballerina project. Then, once you build the project with the `ballerina build`
+command, you should be able to run the resultant JAR by executing the `java -jar` command.
+
+E.g., The `Ballerina.toml` content.
+Change the path to the JDBC driver appropriately.
+
+```toml
+[project]
+org-name= "sample"
+version= "0.1.0"
+
+[platform]
+target = "java8"
+
+    [[platform.libraries]]
+    artafactId = "h2"
+    version = "1.4.200"
+    path = "/path/to/com.h2database.h2-1.4.200.jar"
+    groupId = "com.h2database"
+    modules = ["samplemodule"]
+``` 
+
+Else, if you're trying to run a single BAL file, you can copy the JDBC driver into the `${BALLERINA_HOME}/bre/lib` directory and 
+run the BAL file by executing the `ballerina run` command.
+
+### Client
+To access a database, you must first create a 
+[jdbc:Client](https://ballerina.io/learn/api-docs/ballerina/api-docs/java.jdbc/clients/Client.html) object. 
+The examples for creating a JDBC client can be found below.
```

Comment:
Shall we point to the BBE? Most of the other Module.md files we have pointed to the BBE

Reply:
For this case, I want to provide a link for the Client object definition and also providing the example here similar to BBE. And hence for this case, linking BBE seems to be not relevant.